src/main/kotlin/com/terraformation/backend/tracking/db/Queries.kt

```diff
@@ -26,9 +33,98 @@ fun latestObservationForSubzoneField(observationIdField: Field<ObservationId?>)
               // Prefer results from the same observation as the `observationId` if it exists.
               // True is considered greater than false, so sorting by an "=" expression in
               // descending order means the matches come first.
-              OBSERVATION_ID.eq(observationIdField).desc(),
+              OBSERVATIONS.ID.eq(observationIdField).desc(),
```

Comment:
This is a fix to a bug that showed up when I started adding the `observation_requested_subzones`​ to the tests